lib/tasks/analytics.rake

```diff
@@ -38,6 +60,8 @@ namespace :analytics do
     analytic.closed_cases_count = jurisdiction_monitorees.monitoring_closed_with_purged.count
     analytic.open_cases_count = jurisdiction_monitorees.monitoring_open.count
     analytic.non_reporting_monitorees_count = jurisdiction_monitorees.non_reporting.count
+    analytic.monitoree_counts = all_monitoree_counts(jurisdiction_monitorees)
```

Comment:
This is only saving the monitoree_counts for the _immediate_patients_ for the jurisdiction, you're not adjusting the monitoree counts for the monitorees in the child jurisdicitons. You'll have to add lines similar to these in the add_analytic_to_parent function so that these counts bubble up

Reply:
thanks for catching this, will fix!